### PR TITLE
Improved definitions for awesome-notifications

### PR DESCRIPTION
**Pull request by seebeen**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:

- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>

### CODE REVIEW

File: types/awesome-notifications/index.d.ts [hunk 1]
```diff
@@ -85,12 +85,27 @@ declare namespace AWN {
     }
 
     class Elem {
+        beforeInsert(): void;
+        afterInsert(): void;
         insert(): Elem;
+        replace(el?: Elem): Elem;
```
Comment (iisaduan) [inline types/awesome-notifications/index.d.ts:91]:
```suggestion
        replace(el: Elem): Elem;
```
Is this parameter optional? [package code](https://github.com/f3oall/awesome-notifications/blob/2fd96e6215c7e14bfdcdecd23aea1864e5265c58/src/elem.js#L23)

**Review by iisaduan — CHANGES_REQUESTED** [review]
Please address above comment and then I can approve

File: types/awesome-notifications/index.d.ts (hunk 1 above)
Comment (seebeen) [reply types/awesome-notifications/index.d.ts]: You are right. I changed it.

**Review by iisaduan — APPROVED** [review]
LGTM!

**Comment by seebeen** [comment]
Ready to merge

**Comment by cbn-falias** [comment]
@seebeen Unfortunately there's a regression with the new typings.
The functions `tip`, `info`, `success`, `warning`, `alert` are still returning the `HTMLDivElement`.

https://github.com/f3oall/awesome-notifications/blob/2fd96e6215c7e14bfdcdecd23aea1864e5265c58/src/index.js#L16

**Pull request merged**